11.coreboot.u-root.systemboot/README.md

```diff
@@ -43,7 +43,39 @@ For example, you may want to include static builds of `kexec` or `flashrom`, tha
 
 ## Building a suitable Linux kernel
 
-TODO
+TODO add sample `.config`.
+
+You need a relatively recent kernel. Ideally a kernel 4.16, to have support for VPD variables, but a 4.11 can do the job too, if you don't care about boot entries and want "brute-force" booting only.
+
+We will build a kernel with the following properties:
+
+* small enough to fit most flash chips
+* that can run Go programs (mainly futex and epoll support)
+* with the relevant storage and network drivers
+* with kexec support, so it can boot a new kernel
+* with kexec signature verification disabled (optional)
+* with devtmpfs enabled, since we don't use udev
+* XZ support to decompress the embedded initramfs
+* VPD (Vital Product Data) (optional)
+* TPM support (optional)
+* embed the u-root initramfs
+* and last but not least, "linuxboot" as default host name :)
+
+
+Steps:
+* create a minimal configuration, see https://tiny.wiki.kernel.org
```

Comment:
Please no URLs. They go bad amazingly fast. And for this case, you literally only need a very small .config and let the menuconfig do the rest of the default sets.

Reply:
I can mention `make tinyconfig`, but that could go stale as fast as the wiki URL. I think that wiki URLs are in general better than not having them, because even if stale, you can 1) check how old the docs are, and act accordingly (i.e. google search -> kernel minimal config), and 2) use the URL to know what to search in case the URL changes.
However I can remove it and simply mention `make tinyconfig`

Reply:
actually - what about links in footnotes? They would not be a prominent part of the text, but still readily available